example/src/Utilities/PaymentMethodsProvider.js

```diff
@@ -14,36 +14,39 @@ export const usePaymentMethods = () => {
   return context;
 };
 
-class PaymentMethodsProvider extends Component {
-  state = {
+const PaymentMethodsProvider = (props) => {
+  const [state, setState] = useState({
     config: DEFAULT_CONFIGURATION,
-    paymentMethods: null,
-  };
+    paymentMethods: undefined,
+  });
```

Comment:
There is nothing wrong doing this, but the rule of thumb is that `useState` ideally should manipulate only one type of object. There is no problem creating multiple states.

In this case, it is better to:
```jsx
const [config, setConfig] = useState(DEFAULT_CONFIGURATION);
const [paymentMethods, setPaymentMethods]= useState(undefined);
```

Reply:
Fixed on [c5caa73](https://github.com/Adyen/adyen-react-native/pull/156/commits/c5caa7340d01274755a3e945975441296dcdb11e)